Edits to getting started running on public clouds quickstarts (#208)
pinot-contrib/pinot-docs: basics/getting-started/public-cloud-examples/aws-quickstart.md

---
description: >-
  This quick start guide helps you get started running Pinot on Amazon Web Services (AWS).
---

# Running on AWS

> In this quick start guide you will set up a Kubernetes Cluster on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/)

Reviewer: ```suggestion
In this quickstart guide, you will set up a Kubernetes Cluster on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/)
```

eksctl create addon --name aws-ebs-csi-driver --cluster pinot-quickstart --service-account-role-arn arn:aws:iam::$(aws sts get-caller-identity --query Account --output text):role/AmazonEKS_EBS_CSI_DriverRole --force
```

> Use this command to monitor the cluster status:

Reviewer: ```suggestion
Use the following command to monitor the cluster status:
```

## **4. Connect to an existing cluster**

> Run this command to get the credential for the cluster **pinot-quickstart** that you just created:

Reviewer: ```suggestion
Run the following command to get the credential for the cluster **pinot-quickstart** that you just created:
```

```
EKS_CLUSTER_NAME=pinot-quickstart
aws eks update-kubeconfig --name ${EKS_CLUSTER_NAME}
```

To verify the connection, run:

```
kubectl get nodes
```

## 5. Pinot Quickstart

> Follow this [Kubernetes QuickStart](../kubernetes-quickstart.md) to deploy your Pinot Demo.

Reviewer: ```suggestion
Follow this [Kubernetes QuickStart](../kubernetes-quickstart.md) to deploy your Pinot demo.
```
